src/sfdt/canUseListCondition.ts

```diff
@@ -105,12 +105,13 @@ export function canUseListCondition(block, name) {
 	// represent block is not list
 	const blockParagraphFormat = get(normalizedBlock, 'paragraphFormat');
 	const listFormat = get(blockParagraphFormat, 'listFormat');
+
 	if (!listFormat) {
-		return false;
+		return false
 	}
 
-	if (listFormat && !isEmpty(listFormat)) {
-		return true;
+	if (listFormat && isEmpty(listFormat)) {
+		return false;
```

Comment:
seems like you need a few more unit tests, this is an example of error that unit test should catch